pkgs/development/tools/libsigrok/default.nix

```diff
@@ -19,16 +20,17 @@
 
 stdenv.mkDerivation rec {
   pname = "libsigrok";
-  version = "0.5.2";
+  version = "unstable-2023-11-21";
 
-  src = fetchurl {
-    url = "https://sigrok.org/download/source/${pname}/${pname}-${version}.tar.gz";
-    sha256 = "0g6fl684bpqm5p2z4j12c62m45j1dircznjina63w392ns81yd2d";
+  src = fetchgit {
+    url = "git://sigrok.org/${pname}";
+    rev = "201bd256a8aa85035423023914889fe9f5db52db";
+    hash = "sha256-A1gH07bmNeiFM/yTQ6fE75mrMhGHC+kps7zy6zI51E4=";
```

Comment:
You have updated the version and the commit message, but not the source `rev` and `hash`, which are the same as for the 2023-10-18 version ([rev](https://sigrok.org/gitweb/?p=libsigrok.git;a=commit;h=201bd256a8aa85035423023914889fe9f5db52db)).